lib/util.js

```diff
@@ -36,12 +38,22 @@ export const unescapeHtml = s => {
 
 export const parseRGBA = obj => `rgba(${obj.r},${obj.g},${obj.b},${obj.a})`
 
-export const getState = morph.compose(
+export const getSavedSettings = morph.compose(
```

Comment:
Can we change this to `getSettings` (and `getPresets` accordingly)? The reason being I like the parallelism between `getX` and `saveX`